weaviate/gql/multi_get.py

```diff
@@ -0,0 +1,70 @@
+"""
+GraphQL `Get` command.
+"""
+
+from typing import List
+from weaviate.gql.filter import (
+    GraphQL,
+)
+from weaviate.connect import Connection
+from .get import GetBuilder
+
+
+class MultiGetBuilder(GraphQL):
+    """
+    GetBuilder class used to create GraphQL queries.
+    """
+
+    def __init__(self, get_builder: List[GetBuilder], connection: Connection):
+        """
+        Initialize a MultiGetBuilder class instance.
+
+        Parameters
+        ----------
+        get_builder : list of GetBuilder
+            GetBuilder objects for a single request each.
+        connection : weaviate.connect.Connection
+            Connection object to an active and running Weaviate instance.
+
+        Examples
+        --------
+        To create a 'multi_get' object using several 'get' request at the same time use:
+
+        >>>    client.query.multi_get(
+        ... [
+        ...    client.query.get("Ship", ["name"]).with_alias("one"),
+        ...    client.query.get("Ship", ["size"]).with_alias("two"),
+        ...    client.query.get("Person", ["name"])
+        ... ]
+        with_alias() needs to be used if the same 'class_name' is used twice during the same 'multi_get' request.
+
+        Raises
+        ------
+        TypeError
+            If 'get_builder' is of wrong type.
+        """
+
+        super().__init__(connection)
+        if not isinstance(get_builder, List):
+            raise TypeError(f"get_builder must be of type List but was {type(get_builder)}")
+        for get in get_builder:
```

Comment:
Maybe it would be a good idea to check if there are multiple `GetBuilder`s for the same class. Note that one of the `GetBuilder`s with the same class can be without an alias.

A good way to check it is to have an initial empty `dict` and when you iterate through the builders you add stuff to it. if can be of the form:
```python
{
   "<class_name>" : set("<class_name and alias>")
}
```
When you iterate you check if key exists, if not you add it with value a `set()` containing either the `class_name` or the `alias` if one is set. When the key exists you have to check if the `class_name` or `alias` (if it has one, don't check the `class_name` of one has an alias) is in the `set()`, if it is throw error. if not add to the set.

To make your life better without accessing protected properties, you can add a new one to the `GetBuilder`, something like `name`
```python
@property
def name(self) -> str:
   return self._alias if self._alias else self._class_name
```

and use it. instead of checking both `class_name` and `alias`.

Let me know what do you think of this.